cmd/serve.go

```diff
@@ -324,6 +324,13 @@ func (c *ServeCommand) Command(serverService ServerServiceInterface, monitorServ
 			FlagDefault: 3,
 			Required:    true,
 		},
+		{
+			Name:        "enable-default-tenant",
+			Usage:       "This option enables the Default Tenant feature. In the case where multi-tenancy is not required, this options bypasses the tenant resolution by always resolving to the default tenant configured in the database.",
+			OptType:     types.Bool,
+			ConfigKey:   &serveOpts.EnableDefaultTenant,
+			FlagDefault: false,
+		},
```

Comment:
🤔  "enable default tenant" means that when none is provided, it will be resolved to a default one. But the implementation seems to be "disabling multi tenancy". When I reviewed [the ticket](https://stellarorg.atlassian.net/browse/SDP-1100), it was not clear to me if the idea was to ignore/disable any tenant that doesn't have `is_default=true`.

Reply:
Changed to `single-tenant-mode`